Quests/Kanto/CascadeBadgeQuest.lua

```diff
@@ -8,13 +8,17 @@ local game   = require "Libs/gamelib"
 local Quest  = require "Quests/Quest"
 local Dialog = require "Quests/Dialog"
 
-local name        = 'Cascade Badge - Cerulean City'
-local description = 'Get Cerulean City Badge'
-local level       = 20
+local name        = 'Cascade Badge - Cerulean City: '
+local description = 'Get Badge + Ticket SS.Anne'
+local level       = 21
 
 local dialogs = {
-	fossileGuyBeaten = Dialog:new({
-		"Did you get the one you like?"--,
+	BillTicketDone = Dialog:new({
```

Comment:
BillTicketDone is an instance of an object, it starts by a lower case character.